app/Http/routes.php

```diff
@@ -118,3 +118,9 @@
 	'middleware' => 'admin',
 	'uses' => 'SprintsController@delete'
 ]);
+
+Route::post('sprints', [
```

Comment:
Shouldn't this rather be `sprints/connect` or so because `sprints` only might be used for an index at some point.